style/preview.less

```diff
@@ -143,4 +143,8 @@
 			}
 		}
 	}
+
+	&.mobile {
+		width: 100%;
+	}
```

Comment:
In the layout ticket https://phabricator.wikimedia.org/T250796

> At max, **it expands up to 80% of the screen height** and if there is still more content is available to cover then stick preview header(thumbnail, Wikipedia wordmark, close icon) and enable vertical scroll. It ends with a CTA to read the entire article by visiting mobile Wikiped

Currently the height is set by `max-height` with the fixed pixels, I would expect we apply `calc( 80wh )` here to actually get max 80% of the height size when expanded.

Reply:
opps, I think this is out of the scope of this ticket. 

This should be part of the Layout task - let's work on another PR for this 

1. 80% screen height
1. visiting mobile Wikipedia; visiting desktop Wikipedia based on `isTouch`
1. vertical scrolling in expanded mode

Reply:
The rules above should be part of this task. Can be a separate PR if you wish. In which case you should bring the task back to "dev".

The layout task is about the content of the preview. This task is about where it goes on mobile including sizing.

Reply:
I was literally about to create a separate PR for it (https://github.com/wikimedia/wikipedia-preview/tree/T250796-layout-touch) and was wondering if it should point to this PR or master...  so this simplifies things! Let me merge with this PR and push changes now then

Reply:
![6519665F-4BAF-4C08-86A8-43032F5BD43A](https://user-images.githubusercontent.com/2560096/86901324-bfcef080-c10c-11ea-944c-80fa01ddba8c.png)
![66B87BBA-81C2-4893-98CB-DB8F85BE4296](https://user-images.githubusercontent.com/2560096/86901336-c3fb0e00-c10c-11ea-90b6-cdb7f3f9ffbe.png)